CMakeLists.txt

```diff
@@ -142,23 +142,24 @@ option(NGRAPH_DEX_ONLY "Build CPU DEX without codegen" FALSE)
 option(NGRAPH_CODE_COVERAGE_ENABLE "Enable code coverage data collection" FALSE)
 option(NGRAPH_LIB_VERSIONING_ENABLE "Enable shared library versioning" FALSE)
 option(NGRAPH_PYTHON_BUILD_ENABLE "Enable build nGraph python package wheel" FALSE)
-option(NGRAPH_DISTRIBUTED_MLSL_ENABLE "Add distributed MLSL mode for CPU only backend" FALSE)
-option(NGRAPH_DISTRIBUTED_OMPI_ENABLE "Add distributed Open-MPI mode for all backend" FALSE)
 option(NGRAPH_PLAIDML_ENABLE "Enable the PlaidML backend" ${PLAIDML_FOUND})
+option(NGRAPH_DISTRIBUTED_ENABLE "Enable distributed training using MLSL/OpenMPI" OFF)
 
-if (NGRAPH_GPUH_ENABLE)
-    set(NGRAPH_GPU_ENABLE TRUE)
+if (NGRAPH_DISTRIBUTED_ENABLE)
+    if ("${NGRAPH_DISTRIBUTED_ENABLE}" STREQUAL "MLSL")
+        set(NGRAPH_DISTRIBUTED_MLSL_ENABLE TRUE)
```

Comment:
This needs to check if ANY backend other than the CPU is enabled then it is an error. Since MLSL is only valid if the CPU backend is enabled then it must be the only backend enabled. Does the interpreter support MLSL? Anyway, need to check for backends that will not support distributed if MLSL is enabled.